Classes/Report/AccessFilterPluginInstalledStatus.php

```diff
@@ -94,27 +96,19 @@ protected function checkPluginInstallationStatus(SolrService $solrConnection
         $status = null;
 
         if (!$this->isPluginInstalled($solrConnection)) {
-            $message = '<p style="margin-bottom: 10px;">
-			EXT:solr comes with a plugin for the Apache Solr server to
-			ensure TYPO3 access restrictions are enforced for search
-			results, thus allowing visitors to only see results they are
-			allowed to see. The plugin is not installed on the following
-			server. It is recommended to install the plugin if you are using
-			frontend user group access restrictions for your site.</p>';
-
-            $message .= '<p>Affected Solr server:</p>
-				<ul>'
-                . '<li>Host: ' . $solrConnection->getHost() . '</li>'
-                . '<li>Port: ' . $solrConnection->getPort() . '</li>'
-                . '<li>Path: ' . $solrConnection->getPath() . '</li>
-				</ul>';
-
-            $message .= $this->getPluginDownloadMessage();
-
-            $status = GeneralUtility::makeInstance('TYPO3\\CMS\\Reports\\Status',
+            $standaloneView = GeneralUtility::makeInstance(StandaloneView::class);
+            $standaloneView->setTemplatePathAndFilename(
```

Comment:
You could give this one a (set of) templateRootPath and then call `render($virtualAction)` - that would allow integrators to replace the template. Same for the case below :)